netlify-cms.config.yml

```
@@ -95,3 +95,91 @@ collections:
     label: Past
 
   preview_path: api/preview?page=about&ref=cms/team/{{slug}}
+
+
+- name: history
+
+  folder: history
+  extension: md
+  format: yaml-frontmatter
+
+  media_folder: '/images/history'
+
+  create: true
+  delete: true
+
+  label: 🕰 History
+  label_singular: milestone
+  summary: '{{date}} - {{title}}'
+
+  fields:
+  - name: date
+    label: Date
+    widget: datetime
+    format: YYYY-MM-DD
+    date_format: YYYY-MM-DD
+    time_format: ''
+    default: ''
+    hint: >
+      All milestones on the page are sorted in chronologic order.
+      The day is not displayed, therefore it doesn't have to be correct.
+    picker_utc: true
+  - name: title
+    label: Milestone
+    widget: string
+    hint: >
+      The title for the milestone
+  - name: body
+    widget: markdown
+    buttons: [bold, italic, link, quote, bulleted-list, numbered-list]
+    label: Description
+    hint: >
+      Try to keep this concise, it should be approximately 2-3 lines on the website.
+  - name: image
+    label: Image
+    widget: object
+    summary: 'Image: {{alt}}'
+    hint: >
+      A good illustration of what the milestone is about.
+      Despite it being optional, having illustrations is preferable.
+    fields:
+      - name: src
+        label: Source
+        widget: image
+        allow_multiple: false
+        required: false
+        hint: >
+          Provide a link to the image or upload as a file.
+      - name: alt
+        widget: string
+        required: false
+        label: Alternative text
+        hint: >
+          Description of what is actually displayed on the image.
+          It helps tech-assistive users who can't see the screen understand
+          what is on the picture.
+  - name: link
+    label: Link
+    hint: >
+      The link to a relevant resource
+    widget: object
+    fields:
+      - name: href
+        label: URL
+        widget: string
+      - name: caption
+        label: Caption
+        widget: string
```

I read about the difference between _caption_ and _label_, I misused the _caption_. When _caption_ means something that describes a figure right next to it in whatever context, the _label_ is a kind of caption that must be short. We prefer _label_ than 🙂

```suggestion
      - name: label
        label: Label
        widget: string
```